### PR TITLE
docs(aio):fix numbering of the three ways to access hero details

### LINKED ISSUE #20628: Numbering bug in documentation of Angular 5
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[x ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior

Numbered list is shown in a wrong way.

1. Foo 1. Bar 1. Baz

## Expected behavior

1. Foo 2. Bar 3. Baz

## Minimal reproduction of the problem with instructions

1. Open https://angular.io/tutorial/toh-pt5
2. Go to "Navigating to hero details" section

## What is the motivation / use case for changing the behavior?

This is just a small bug in the documentation.

## Screenshot
kep

### PR DESCRIPTION
**Pull request by benbraou**
Fixes #20628

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #20628

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: aio/content/tutorial/toh-pt5.md [hunk 1]
```diff
@@ -276,8 +276,8 @@ At the moment the `HeroDetailsComponent` is only visible at the bottom of the `H
 
 The user should be able to get to these details in three ways.
 1. By clicking a hero in the dashboard.
-1. By clicking a hero in the heroes list.
-1. By pasting a "deep link" URL into the browser address bar that identifies the hero to display.
+2. By clicking a hero in the heroes list.
+3. By pasting a "deep link" URL into the browser address bar that identifies the hero to display.
```
Comment (gkalpak) [inline aio/content/tutorial/toh-pt5.md]: Using the same number (e.g. `1`) works as well for creating number lists (and makes it easier to re-order or add/remove items). (As you can see in the preview, changing to `1,2,3` doesn't fix the actual issue.
Maybe there needs to be an empty line above the list. Can you make this change and see if it helps?

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/content/tutorial/toh-pt5.md (hunk 1 above)
Comment (benbraou) [reply aio/content/tutorial/toh-pt5.md]: Thanks for having took the time to review.  I have checked the readme for Angular documentation project and see now how to preview the changes...
Indeed, an empty line is missing.

**Comment by benbraou** [comment]
@gkalpak are you ok with the change now?

**Review by gkalpak — APPROVED** [review]
Very OK :smiley:

**Pull request closed without merging**